docs/development/build-instructions-gn.md

```diff
@@ -90,29 +90,35 @@ $ gclient sync -f
 
 ## Building
 
+There are three different build configurations you can potentially set up - `Debug`, `Testing`, and `Release`.
```

Comment:
Should we explain what the difference is between these? How should people choose?

Reply:
this is still a relevant, unaddressed comment